dags/looker.py

```diff
@@ -217,5 +217,23 @@
         **airflow_gke_prod_kwargs,
     )
 
+    delete_outdated_branches = GKEPodOperator(
+        task_id="delete_outdated_branches",
+        arguments=[
+            "python",
+            "-m",
+            "looker_utils.main",
+            "delete-branches",
+            "--n_days 180"
```

Comment:
Since this combination of parameter and value is being passed as a single discrete argument, does it maybe need to use `=` instead of a space?
```suggestion
            "--n_days=180"
```

Reply:
```suggestion
            "--inactive_days=180"
```